core/iwasm/interpreter/wasm_runtime.h

```diff
@@ -320,7 +320,9 @@ struct WASMModuleInstance {
 
     /* Default WASM operand stack size */
     uint32 default_wasm_stack_size;
-    uint32 reserved[3];
+    /* Use one of reserved element to implement disable_boundary_check */
+    uint32 disable_boundary_check;
```

Comment:
Consider to move this filed to WASMModule, don't use this filed to restore reserved bytes.